_data/navigation.js

```diff
@@ -194,6 +194,7 @@ module.exports = {
           { title: 'Introduction', url: '/docs/chainlink-keepers/introduction/' },
           { title: 'Making Compatible Contracts', url: '/docs/chainlink-keepers/compatible-contracts/' },
           { title: 'Register Upkeep for a Contract', url: '/docs/chainlink-keepers/register-upkeep/' },
+          { title: 'Patterns and Best Practices', url: '/docs/chainlink-keepers/best-practices/' },
```

Comment:
Register Upkeep's "What's Next" section goes straight to Network Overview instead of Patterns and Best Practices"